b2/console_tool.py

```diff
@@ -344,6 +344,25 @@ def run(self, args):
         return 0
 
 
+class GetFileInfoAndDeleteFileVersion(Command):
```

Comment:
The command-line parser we have requires that optional arguments be last.  What we have now is `b2 delete_file_version <fileName> <fileId>`.

I could rename the new command `delete_file_version_by_id`.